Uchu.StandardScripts/BlockYard/BlockYardWarning.cs

```diff
@@ -0,0 +1,20 @@
+using System.Threading.Tasks;
+using Uchu.World.Scripting.Native;
+using Uchu.World.Social;
+
+namespace Uchu.StandardScripts.BlockYard
+{
+    [ZoneSpecific(1150)]
+    public class BlockYardWarning : NativeScript
+    {
+        public override Task LoadAsync()
+        {
+            Listen(Zone.OnPlayerLoad, async player =>
```

Comment:
I like that this message is displayed to the user rather than just blindly complete the missions for the user. `/testmap` requires Mythran permissions though, which not everyone will grant themselves.

Reply:
Tested it and it actually doesn't matter that much. Didn't realize the skipping missions happens in Avant Gardens, not Block Yard.

Reply:
There's no other way to get out of Block Yard besides that command, so unless we fix that launchpad or disable the one in AG I don't think this can be improved

Reply:
I guess all we can hope is that players don't go to Block Yard. The reason to go no longer really exists because of skipping the missions in Avant Gardens.